wrappers/services.go

```diff
@@ -1575,22 +1575,43 @@ type RestartServicesFlags struct {
 }
 
 // Restart or reload services; if reload flag is set then "systemctl reload-or-restart" is attempted.
-func RestartServices(svcs []*snap.AppInfo, flags *RestartServicesFlags, inter interacter, tm timings.Measurer) error {
+// The `explicitServices` list can be null to indicate the the action was
+// requested on the snap as a whole, otherwise it needs to contain the names of
+// the services which were explicitly requested to be restarted.
+func RestartServices(svcs []*snap.AppInfo, explicitServices []string,
```

Comment:
it seems we need new unit tests for the case when explicitServices is not nil

Reply:
Added

Reply:
need one more unit test case I think